README.md

```diff
@@ -2,38 +2,31 @@
 
 [![GitHub Build Status](https://github.com/cisagov/terraform-state-read-role-tf-module/workflows/build/badge.svg)](https://github.com/cisagov/terraform-state-read-role-tf-module/actions)
 
-This is a generic skeleton project that can be used to quickly get a
-new [cisagov](https://github.com/cisagov) [Terraform
-module](https://www.terraform.io/docs/modules/index.html) GitHub
-repository started.  This skeleton project contains [licensing
-information](LICENSE), as well as [pre-commit
-hooks](https://pre-commit.com) and
-[GitHub Actions](https://github.com/features/actions) configurations
-appropriate for the major languages that we use.
-
-See [here](https://www.terraform.io/docs/modules/index.html) for more
-details on Terraform modules and the standard module structure.
+This is a Terraform module for creating an IAM role and policy that can
+read Terraform state objects from a specified S3 bucket.  It also creates
+a policy that allows the role to be assumed from a specified list of AWS
+account IDs.
 
 ## Usage ##
 
 ```hcl
 module "example" {
   source = "github.com/cisagov/terraform-state-read-role-tf-module"
-
-  aws_region            = "us-west-1"
-  aws_availability_zone = "b"
-  subnet_id             = "subnet-0123456789abcdef0"
-
-  tags = {
-    Key1 = "Value1"
-    Key2 = "Value2"
+  providers = {
+    aws       = aws
+    aws.users = aws.users
   }
+
+  account_ids = ["111111111111"]
+  role_name = "ReadTerraformStateReadRoleTFModuleTerraformState"
+  terraform_state_bucket_name = "cisa-cool-terraform-state"
```

Comment:
Maybe make this match the tfvar example:

```suggestion
  terraform_state_bucket_name = "example-terraform-state-bucket"
```

Reply:
I think we decided long ago that the bucket name is not particularly sensitive.  If that were to change, we would have [a fair amount of files to change](https://github.com/search?q=org%3Acisagov+cisa-cool-terraform-state&type=code).

I'm going to keep the bucket name as is, unless you strongly disagree with that plan.